docs/changelog.txt

```diff
@@ -67,6 +67,11 @@ Template for new versions:
 
 ## Removed
 
+# 50.10-r2
+
+## New Plugins
+- `tubefill`: (reinstated) replenishes mined-out adamantine
+
```

Comment:
We'll create the 50.10-r2 header when we tag the release. For now, the `tubefill` entry should go under "New Tools" under the "Future" heading.